### PR TITLE
Card + Logo

### PR DESCRIPTION
**Pull request by Arman-Akther**
- cards made on index.html.erb
- _card.scss component created and completed
- logo created and animated on index.html.erb ~ given to pedro to add to navbar
- _logo.scss component created and completed

### CODE REVIEW

**Review by cazbacelar — APPROVED** [review]
LGTM

File: app/views/pets/index.html.erb [hunk 1]
```diff
@@ -1,12 +1,45 @@
...
+          </div>
+        <% end %>
+      </div>
+    </div>
+  <% end %>
+</div>
+
+
+
 <ul>
```
Comment (LucasNseyep) [inline app/views/pets/index.html.erb:38]: You can remove this whole UL since the above is the equivalent but with images

**Review by LucasNseyep — APPROVED** [review]
One small problem to resolve but otherwise looks good.

File: app/views/pets/index.html.erb [hunk 2]
```diff
@@ -1,12 +1,45 @@
...
   <% @pets.each do |pet| %>
   <%# LINK THROUGH THE PHOTO IN FINAL ONE %>
-  <li><%= link_to pet.name, pet_path(pet) %></li>
+  <li>
+    <%= link_to pet.name, pet_path(pet) %>
+  </li>
   <% end %>
 </ul>
```
Comment (LucasNseyep) [inline app/views/pets/index.html.erb:45]: You can remove this whole `ul` since the above is the equivalent but with images

**Review by LucasNseyep — APPROVED** [review]
LGTM apart from the `ul` you can remove

**Pull request merged**